### PR TITLE
Decouple Sana logic from homepage

### LINKED ISSUE #55: Pull out editor as separate Django app
The `webapp` app should just manage the static pages while the `editor` app manages the API endpoints.

### PR DESCRIPTION
**Pull request by Trinovantes**
- Renamed `webapp` app to `homepage`
- Created `editor` app

Closes #55

### CODE REVIEW

**Comment by Kapin** [comment]
I'm assuming these will share the same database? We need to be able to reference users from procedures for ownership purposes.

**Comment by Trinovantes** [comment]
Same database. You can reference by importing the `User` model from `django.contrib.auth.models`

**Comment by Trinovantes** [comment]
I just noticed I forgot to move `editor.html`. Let me make 1 more commit before reviewing this PR

File: src/editor/tests/test_user.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# from django.test import TestCase
```
Comment (ConnorCimowsky) [inline src/editor/tests/test_user.py]: Any reason this file is commented out?

**Comment by ConnorCimowsky** [comment]
One question, otherwise LGTM as long as you've tested it locally.

File: src/editor/tests/test_user.py (hunk 1 above)
Comment (Trinovantes) [inline src/editor/tests/test_user.py]: It's going to be moved to a `registration` app that I'm working on. I don't want the merge conflict or a temporary broken test.

Comment (Trinovantes) [inline src/editor/tests/test_user.py]: Actually I don't even see why I added this file in the first place...

Comment (ConnorCimowsky) [inline src/editor/tests/test_user.py]: Let me know what you decide and I'll re-review. :stuck_out_tongue:

**Comment by ConnorCimowsky** [comment]
LGTM

**Comment by Kapin** [comment]
Same

**Pull request merged**